app/routes/crate.js

```diff
@@ -1,39 +0,0 @@
-import Ember from 'ember';
-import Version from 'cargo/models/version';
-
-export default Ember.Route.extend({
-    model(params) {
-        var parts = params.crate_id.split('/');
-        var crate_id = parts[0];
-        var version = null;
-        if (parts.length > 1 && parts[1].length > 0) {
-            version = parts[1];
-        }
-
-        var crate = this.store.find('crate', crate_id).catch((e) => {
-          if (e.status === 404) {
-            this.controllerFor('application').set('nextFlashError', 'No crate named: ' + params.crate_id);
```

Comment:
I think something around here made some routes return weird errors instead of flash messages perhaps? For example going to these two routes yield JS exceptions:

* http://localhost:4200/crates/libc/0.3.4
* http://localhost:4200/crates/wut